### PR TITLE
New games collection

### PR DESCRIPTION
**Pull request by parw8649**
**What was resolved?**

- Updated games & event functionality to update recycler view
- Added layout for the Hosted games page
- Added layout for the single Hosted game page
- Added functionality logic for Hosted games, yet to implement it completely

### CODE REVIEW

**Review by bKondo — APPROVED** [review]

**Comment by BorisMarin8004** [comment]
@parw8649 did you fix the background change issue on game selection?

**Comment by parw8649** [comment]
@BorisMarin8004 I haven’t added the background fix yet but I will be adding it along with the hosted game logic in the next PR

**Comment by parw8649** [comment]
> @BorisMarin8004 I haven’t added the background fix yet but I will be adding it along with the hosted game logic in the next PR

@BorisMarin8004 I haven’t added the background fix yet but I will be adding it along with the hosted game logic in the next PR

File: Android/app/src/main/java/com/example/boardgamesocial/DataViews/Adapters/DataClsAdapter.java [hunk 1]
```diff
@@ -72,12 +72,27 @@ public void onBindViewHolder(VH holder, int position) {
...
             }
-        }
+
+            objectList.removeIf(obj -> {
+                boolean exists = newObjects.contains(obj);
+                if(!exists)
+                    notifyDataSetChanged();
+                return !exists;
+            });
+        }*/
```
Comment (BorisMarin8004) [inline Android/app/src/main/java/com/example/boardgamesocial/DataViews/Adapters/DataClsAdapter.java:93]: I think for removal we should use something like this 
```java
mDataset.remove(position);
notifyItemRemoved(position);
notifyItemRangeChanged(position, mDataSet.size());
```
Here is a link to [StackOverflow](https://stackoverflow.com/questions/26076965/android-recyclerview-addition-removal-of-items)

**Review by BorisMarin8004 — APPROVED** [review]
Nice 👍

**Review by keyoni — APPROVED** [review]

**Pull request merged**